train.py

```diff
@@ -484,6 +493,14 @@ def main(args):  # pylint: disable=redefined-outer-name
     else:
         optimizer_st = None
 
+    if c.apex_amp_level:
+        # pylint: disable=import-outside-toplevel
+        from apex import amp
```

Comment:
nope, apex is only loaded / required if the respective setting is active in the config as you can see